### ISSUE TITLE
TypeError: Cannot redefine property: default

### ISSUE THREAD

**Issue by jt-helsinki** [terrestris/base-util#283]
When building the application using Webpack the application the `Logger.ts` (`Logger.js`) file causes the following error: 

```
TypeError: Cannot redefine property: default
(anonymous function)
node_modules/@terrestris/base-util/dist/Logger.js:10
   7 |     o[k2] = m[k];
   8 | }));
   9 | var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
> 10 |     Object.defineProperty(o, "default", { enumerable: true, value: v });
  11 | }) : function(o, v) {
  12 |     o["default"] = v;
  13 | });

View compiled
./node_modules/@terrestris/base-util/dist/Logger.js.__importStar
node_modules/@terrestris/base-util/dist/Logger.js:18
  15 |     if (mod && mod.__esModule) return mod;
  16 |     var result = {};
  17 |     if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) __createBinding(result, mod, k);
> 18 |     __setModuleDefault(result, mod);
  19 |     return result;
  20 | };
  21 | Object.defineProperty(exports, "__esModule", { value: true });

View compiled
./node_modules/@terrestris/base-util/dist/Logger.js
node_modules/@terrestris/base-util/dist/Logger.js:22
  19 |     return result;
  20 | };
  21 | Object.defineProperty(exports, "__esModule", { value: true });
> 22 | var log = __importStar(require("loglevel"));
  23 | var logger = log.getLogger('app-logger');
  24 | // TODO Make configurable via app.config.js, NODE_ENV or similiar.
  25 | logger.setLevel('DEBUG');

```

The Webpack scripts are based on ejected react-dev scripts and Logger is being imported via **@terrestris
/react-geo**. 

The solution to this is to change `Logger.ts` to import the default import for the `logLevel` package.

`import log from 'loglevel';`

**Comment by jt-helsinki** [comment]
PR with fix https://github.com/terrestris/base-util/pull/284

**Comment by hwbllmnn** [comment]
Fix is included in 0.2.4, thanks again!

**Comment by jt-helsinki** [comment]
@hwbllmnn This package version will need to be updated in react-geo too. :)
